test/libsolidity/syntaxTests/userDefinedValueType/recursive_err.sol

```diff
@@ -0,0 +1,4 @@
+type MyInt1 is MyInt2;
+type MyInt2 is MyInt1;
+// ----
+// TypeError 8657: (15-21): The underlying type for a user defined value type has to be an elementary value type.
```

Comment:
This is a bad restriction. We can of course keep this for now, but we *do* plan to lift this eventually, right :-)?